backend/app/expenses/routes.py

```
@@ -460,3 +460,52 @@
         raise HTTPException(status_code=404, detail=str(e))
     except Exception as e:
         raise HTTPException(status_code=500, detail="Failed to fetch analytics")
+
+
+# Debug endpoint (remove in production)
+@router.get("/expenses/{expense_id}/debug")
+async def debug_expense(
+    group_id: str,
+    expense_id: str,
+    current_user: Dict[str, Any] = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)